lib/lightning_web/live/user_live/form_component.html.heex

```diff
@@ -103,27 +103,28 @@
       <div class="hidden sm:block" aria-hidden="true">
         <div class="py-5"></div>
       </div>
-      <div class="flex items-start">
-        <div class="flex items-center h-5">
-          <%= checkbox(f, :disabled,
-            class:
-              "focus:ring-primary-500 h-4 w-4 text-primary-600 border-secondary-300 rounded"
-          ) %>
-        </div>
-        <div class="ml-3 text-sm">
-          <%= error_tag(f, :disabled,
-            class:
-              "mt-1 focus:ring-primary-500 focus:border-primary-500 block w-full shadow-sm sm:text-sm border-secondary-300 rounded-md"
-          ) %>
-          <%= label(f, :disabled, class: "font-medium text-secondary-700") do %>
-            Disabled?
-          <% end %>
-          <p class="text-secondary-500">
-            A disabled user will not be allowed to login.
-          </p>
+      <%= if @can_disable_users do %>
```

Comment:
I am not sure this works. Can you access the `can_disable_users` without assigning it in the `update/2` function ? Is there any liveview test to capture this ?